augly/image/transforms.py

```diff
@@ -1310,7 +1309,34 @@ def apply_transform(
             image, factor=self.factor, seed=self.seed, metadata=metadata
         )
 
-
+class Skew(BaseTransform):
```

Comment:
add a newline before this class definition

Reply:
Done !!!